*** ./smali/com/android/internal/telephony/cdma/CDMAPhone.smali	2013-06-06 02:36:26.991001345 +0430
--- /home/sijav/patchrom/nypone/temp/new_smali/framework.jar.out/./smali/com/android/internal/telephony/cdma/CDMAPhone.smali	2013-06-06 02:35:23.359000286 +0430
***************
*** 1103,1108 ****
--- 1103,1112 ----
  # virtual methods
  .method public acceptCall()V
      .locals 1
+     .annotation build Landroid/annotation/MiuiHook;
+         value = .enum Landroid/annotation/MiuiHook$MiuiHookType;->CHANGE_CODE:Landroid/annotation/MiuiHook$MiuiHookType;
+     .end annotation
+ 
      .annotation system Ldalvik/annotation/Throws;
          value = {
              Lcom/android/internal/telephony/CallStateException;
***************
*** 1110,1115 ****
--- 1114,1123 ----
      .end annotation
  
      .prologue
+     const/16 v0, 0xf
+ 
+     invoke-virtual {p0, v0}, Lcom/android/internal/telephony/cdma/CDMAPhone;->removeMessages(I)V
+ 
      iget-object v0, p0, Lcom/android/internal/telephony/cdma/CDMAPhone;->mCT:Lcom/android/internal/telephony/cdma/CdmaCallTracker;
  
      invoke-virtual {v0}, Lcom/android/internal/telephony/cdma/CdmaCallTracker;->acceptCall()V
