*** ./smali/android/preference/PreferenceActivity.smali	2013-07-12 02:23:24.972206607 +0430
--- /home/sijav/patchrom/nypon/temp/new_smali/framework.jar.out/./smali/android/preference/PreferenceActivity.smali	2013-07-12 02:22:16.928204937 +0430
***************
*** 2045,2050 ****
--- 2045,2052 ----
  
      if-lez v17, :cond_e
  
+     invoke-static/range {p0 .. p0}, Landroid/preference/PreferenceActivity$Injector;->setListViewPadding(Landroid/preference/PreferenceActivity;)V
+ 
      new-instance v17, Landroid/preference/PreferenceActivity$HeaderAdapter;
  
      move-object/from16 v0, p0
