*** ./smali/android/preference/PreferenceGroup.smali	2013-06-06 02:36:31.167001414 +0430
--- /home/sijav/patchrom/nypone/temp/new_smali/framework.jar.out/./smali/android/preference/PreferenceGroup.smali	2013-06-06 02:35:27.519000356 +0430
***************
*** 257,262 ****
--- 257,264 ----
      :cond_5
      invoke-virtual {p0}, Landroid/preference/PreferenceGroup;->notifyHierarchyChanged()V
  
+     invoke-virtual {p1, p0}, Landroid/preference/Preference;->setParent(Landroid/preference/PreferenceGroup;)V
+ 
      move v1, v2
  
      goto :goto_0
