*** ./smali/android/preference/PreferenceGroup.smali	2013-07-12 02:23:25.248206612 +0430
--- /home/sijav/patchrom/nypon/temp/new_smali/framework.jar.out/./smali/android/preference/PreferenceGroup.smali	2013-07-12 02:22:17.208204944 +0430
***************
*** 257,262 ****
--- 257,264 ----
      :cond_5
      invoke-virtual {p0}, Landroid/preference/PreferenceGroup;->notifyHierarchyChanged()V
  
+     invoke-virtual {p1, p0}, Landroid/preference/Preference;->setParent(Landroid/preference/PreferenceGroup;)V
+ 
      move v1, v2
  
      goto :goto_0
