*** ./smali/com/android/internal/widget/ScrollingTabContainerView$TabClickListener.smali	2013-06-06 02:36:23.327001284 +0430
--- /home/sijav/patchrom/nypone/temp/new_smali/framework.jar.out/./smali/com/android/internal/widget/ScrollingTabContainerView$TabClickListener.smali	2013-06-06 02:35:18.763000210 +0430
***************
*** 111,115 ****
--- 111,119 ----
  
      .end local v0           #child:Landroid/view/View;
      :cond_1
+     iget-object v4, p0, Lcom/android/internal/widget/ScrollingTabContainerView$TabClickListener;->this$0:Lcom/android/internal/widget/ScrollingTabContainerView;
+ 
+     invoke-static {v4, p1}, Lcom/android/internal/widget/ScrollingTabContainerView$Injector;->onTabViewClick(Lcom/android/internal/widget/ScrollingTabContainerView;Landroid/view/View;)V
+ 
      return-void
  .end method
