*** ./smali/android/view/Window.smali	2013-06-06 02:36:39.419001552 +0430
--- /home/sijav/patchrom/nypone/temp/new_smali/framework.jar.out/./smali/android/view/Window.smali	2013-06-06 02:35:34.647000474 +0430
***************
*** 548,553 ****
--- 548,568 ----
  .method public abstract isFloating()Z
  .end method
  
+ .method public isHardwareAccelerated()Z
+     .locals 1
+ 
+     .prologue
+     invoke-virtual {p0}, Landroid/view/Window;->getWindowManager()Landroid/view/WindowManager;
+ 
+     move-result-object v0
+ 
+     invoke-interface {v0}, Landroid/view/WindowManager;->isHardwareAccelerated()Z
+ 
+     move-result v0
+ 
+     return v0
+ .end method
+ 
  .method public abstract isShortcutKey(ILandroid/view/KeyEvent;)Z
  .end method
  
