*** ./smali/com/android/server/am/ActivityStack.smali	2013-06-06 02:37:25.367002315 +0430
--- /home/sijav/patchrom/nypone/temp/new_smali/services.jar.out/./smali/com/android/server/am/ActivityStack.smali	2013-06-06 02:36:13.635001122 +0430
***************
*** 6226,6231 ****
--- 6226,6234 ----
      .locals 7
      .parameter "r"
      .parameter "globalChanges"
+     .annotation build Landroid/annotation/MiuiHook;
+         value = .enum Landroid/annotation/MiuiHook$MiuiHookType;->CHANGE_CODE:Landroid/annotation/MiuiHook$MiuiHookType;
+     .end annotation
  
      .prologue
      const/4 v3, 0x1
***************
*** 6316,6321 ****
--- 6319,6334 ----
      if-eqz v5, :cond_c
  
      :cond_7
+     iget-object v5, p1, Lcom/android/server/am/ActivityRecord;->info:Landroid/content/pm/ActivityInfo;
+ 
+     iget-object v5, v5, Landroid/content/pm/ActivityInfo;->packageName:Ljava/lang/String;
+ 
+     invoke-static {v5, v0, v1}, Landroid/app/MiuiThemeHelper;->needRestartActivity(Ljava/lang/String;ILandroid/content/res/Configuration;)Z
+ 
+     move-result v5
+ 
+     if-nez v5, :cond_0
+ 
      iget v5, p1, Lcom/android/server/am/ActivityRecord;->configChangeFlags:I
  
      or-int/2addr v5, v0
