*** ./smali/com/android/server/am/AppNotRespondingDialog.smali	2013-06-06 02:37:25.707002321 +0430
--- /home/sijav/patchrom/nypone/temp/new_smali/services.jar.out/./smali/com/android/server/am/AppNotRespondingDialog.smali	2013-06-06 02:36:13.919001127 +0430
***************
*** 300,305 ****
--- 300,317 ----
  
  
  # virtual methods
+ .method getProc()Lcom/android/server/am/ProcessRecord;
+     .locals 1
+     .annotation build Landroid/annotation/MiuiHook;
+         value = .enum Landroid/annotation/MiuiHook$MiuiHookType;->NEW_METHOD:Landroid/annotation/MiuiHook$MiuiHookType;
+     .end annotation
+ 
+     .prologue
+     iget-object v0, p0, Lcom/android/server/am/AppNotRespondingDialog;->mProc:Lcom/android/server/am/ProcessRecord;
+ 
+     return-object v0
+ .end method
+ 
  .method public onStop()V
      .locals 0
  
