*** ./smali/android/app/Notification$Builder.smali	2013-06-06 02:37:16.819002174 +0430
--- /home/sijav/patchrom/nypone/temp/new_smali/framework.jar.out/./smali/android/app/Notification$Builder.smali	2013-06-06 02:36:06.375001002 +0430
***************
*** 44,49 ****
--- 44,51 ----
  
  .field private mContext:Landroid/content/Context;
  
+ .field private mCustomizedIcon:Z
+ 
  .field private mDefaults:I
  
  .field private mDeleteIntent:Landroid/app/PendingIntent;
***************
*** 970,975 ****
--- 972,981 ----
      invoke-virtual {v1, v2}, Ljava/util/ArrayList;->toArray([Ljava/lang/Object;)[Ljava/lang/Object;
  
      :cond_3
+     iget-boolean v1, p0, Landroid/app/Notification$Builder;->mCustomizedIcon:Z
+ 
+     iput-boolean v1, v0, Landroid/app/Notification;->customizedIcon:Z
+ 
      return-object v0
  
      :cond_4
***************
*** 1306,1311 ****
--- 1312,1327 ----
  
      return-object p0
  .end method
+ 
+ .method public setCustomizedIcon(Z)Landroid/app/Notification$Builder;
+     .locals 0
+     .parameter "customizedIcon"
+ 
+     .prologue
+     iput-boolean p1, p0, Landroid/app/Notification$Builder;->mCustomizedIcon:Z
+ 
+     return-object p0
+ .end method
  
  .method public setDefaults(I)Landroid/app/Notification$Builder;
      .locals 0
