*** ./smali/android/os/Vibrator.smali	2013-07-12 02:23:26.276206639 +0430
--- /home/sijav/patchrom/nypon/temp/new_smali/framework.jar.out/./smali/android/os/Vibrator.smali	2013-07-12 02:22:18.172204968 +0430
***************
*** 26,28 ****
--- 26,39 ----
  
  .method public abstract vibrate([JI)V
  .end method
+ 
+ .method public vibrateEx([B)V
+     .locals 0
+     .parameter "params"
+     .annotation build Landroid/annotation/MiuiHook;
+         value = .enum Landroid/annotation/MiuiHook$MiuiHookType;->NEW_METHOD:Landroid/annotation/MiuiHook$MiuiHookType;
+     .end annotation
+ 
+     .prologue
+     return-void
+ .end method
