*** ./smali/com/android/internal/telephony/IIccPhoneBook.smali	2013-06-06 02:36:26.303001333 +0430
--- /home/sijav/patchrom/nypone/temp/new_smali/framework.jar.out/./smali/com/android/internal/telephony/IIccPhoneBook.smali	2013-06-06 02:35:22.683000275 +0430
***************
*** 15,20 ****
--- 15,28 ----
  
  
  # virtual methods
+ .method public abstract getAdnCapacity()I
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
  .method public abstract getAdnRecordsInEf(I)Ljava/util/List;
      .annotation system Ldalvik/annotation/Signature;
          value = {
***************
*** 37,42 ****
--- 45,58 ----
      .annotation system Ldalvik/annotation/Throws;
          value = {
              Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
+ .method public abstract getFreeAdn()I
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
          }
      .end annotation
  .end method
