*** ./smali/com/android/internal/telephony/SMSDispatcher.smali	2013-06-06 02:36:28.227001365 +0430
--- /home/sijav/patchrom/nypone/temp/new_smali/framework.jar.out/./smali/com/android/internal/telephony/SMSDispatcher.smali	2013-06-06 02:35:24.591000307 +0430
***************
*** 7,13 ****
  .annotation system Ldalvik/annotation/MemberClasses;
      value = {
          Lcom/android/internal/telephony/SMSDispatcher$ConfirmDialogListener;,
!         Lcom/android/internal/telephony/SMSDispatcher$SmsTracker;
      }
  .end annotation
  
--- 7,14 ----
  .annotation system Ldalvik/annotation/MemberClasses;
      value = {
          Lcom/android/internal/telephony/SMSDispatcher$ConfirmDialogListener;,
!         Lcom/android/internal/telephony/SMSDispatcher$SmsTracker;,
!         Lcom/android/internal/telephony/SMSDispatcher$Injector;
      }
  .end annotation
  
***************
*** 848,853 ****
--- 849,857 ----
  .method protected dispatchNormalMessage(Lcom/android/internal/telephony/SmsMessageBase;)I
      .locals 14
      .parameter "sms"
+     .annotation build Landroid/annotation/MiuiHook;
+         value = .enum Landroid/annotation/MiuiHook$MiuiHookType;->CHANGE_CODE:Landroid/annotation/MiuiHook$MiuiHookType;
+     .end annotation
  
      .prologue
      const/4 v9, 0x0
***************
*** 897,903 ****
  
      move-result-object v1
  
!     invoke-virtual {v0, v1}, Lcom/android/internal/telephony/WapPushOverSms;->dispatchWapPdu([B)I
  
      move-result v8
  
--- 901,911 ----
  
      move-result-object v1
  
!     invoke-virtual {p1}, Lcom/android/internal/telephony/SmsMessageBase;->getOriginatingAddress()Ljava/lang/String;
! 
!     move-result-object v2
! 
!     invoke-virtual {v0, v1, v2}, Lcom/android/internal/telephony/WapPushOverSms;->dispatchWapPdu([BLjava/lang/String;)I
  
      move-result v8
  
***************
*** 963,970 ****
--- 971,991 ----
  .method protected dispatchPdus([[B)V
      .locals 3
      .parameter "pdus"
+     .annotation build Landroid/annotation/MiuiHook;
+         value = .enum Landroid/annotation/MiuiHook$MiuiHookType;->CHANGE_CODE:Landroid/annotation/MiuiHook$MiuiHookType;
+     .end annotation
  
      .prologue
+     invoke-static {p0, p1}, Lcom/android/internal/telephony/SMSDispatcher$Injector;->checkFireWallForSms(Lcom/android/internal/telephony/SMSDispatcher;[[B)Z
+ 
+     move-result v1
+ 
+     if-eqz v1, :cond_0
+ 
+     :goto_0
+     return-void
+ 
+     :cond_0
      new-instance v0, Landroid/content/Intent;
  
      const-string v1, "android.provider.Telephony.SMS_RECEIVED"
***************
*** 988,1002 ****
  
      invoke-virtual {p0, v0, v1}, Lcom/android/internal/telephony/SMSDispatcher;->dispatch(Landroid/content/Intent;Ljava/lang/String;)V
  
!     return-void
  .end method
  
  .method protected dispatchPortAddressedPdus([[BI)V
      .locals 4
      .parameter "pdus"
      .parameter "port"
  
      .prologue
      new-instance v2, Ljava/lang/StringBuilder;
  
      invoke-direct {v2}, Ljava/lang/StringBuilder;-><init>()V
--- 1009,1036 ----
  
      invoke-virtual {p0, v0, v1}, Lcom/android/internal/telephony/SMSDispatcher;->dispatch(Landroid/content/Intent;Ljava/lang/String;)V
  
!     goto :goto_0
  .end method
  
  .method protected dispatchPortAddressedPdus([[BI)V
      .locals 4
      .parameter "pdus"
      .parameter "port"
+     .annotation build Landroid/annotation/MiuiHook;
+         value = .enum Landroid/annotation/MiuiHook$MiuiHookType;->CHANGE_CODE:Landroid/annotation/MiuiHook$MiuiHookType;
+     .end annotation
  
      .prologue
+     invoke-static {p0, p1}, Lcom/android/internal/telephony/SMSDispatcher$Injector;->checkFireWallForSms(Lcom/android/internal/telephony/SMSDispatcher;[[B)Z
+ 
+     move-result v2
+ 
+     if-eqz v2, :cond_0
+ 
+     :goto_0
+     return-void
+ 
+     :cond_0
      new-instance v2, Ljava/lang/StringBuilder;
  
      invoke-direct {v2}, Ljava/lang/StringBuilder;-><init>()V
***************
*** 1043,1049 ****
  
      invoke-virtual {p0, v0, v2}, Lcom/android/internal/telephony/SMSDispatcher;->dispatch(Landroid/content/Intent;Ljava/lang/String;)V
  
!     return-void
  .end method
  
  .method public abstract dispose()V
--- 1077,1083 ----
  
      invoke-virtual {p0, v0, v2}, Lcom/android/internal/telephony/SMSDispatcher;->dispatch(Landroid/content/Intent;Ljava/lang/String;)V
  
!     goto :goto_0
  .end method
  
  .method public abstract dispose()V
***************
*** 1651,1656 ****
--- 1685,1693 ----
      .parameter "timestamp"
      .parameter "destPort"
      .parameter "isCdmaWapPush"
+     .annotation build Landroid/annotation/MiuiHook;
+         value = .enum Landroid/annotation/MiuiHook$MiuiHookType;->CHANGE_CODE:Landroid/annotation/MiuiHook$MiuiHookType;
+     .end annotation
  
      .prologue
      const/16 v20, 0x0
***************
*** 2165,2171 ****
  
      iget-object v3, v0, Lcom/android/internal/telephony/SMSDispatcher;->mWapPush:Lcom/android/internal/telephony/WapPushOverSms;
  
!     invoke-virtual {v3, v13}, Lcom/android/internal/telephony/WapPushOverSms;->dispatchWapPdu([B)I
  
      move-result v3
  
--- 2202,2210 ----
  
      iget-object v3, v0, Lcom/android/internal/telephony/SMSDispatcher;->mWapPush:Lcom/android/internal/telephony/WapPushOverSms;
  
!     move-object/from16 v0, p2
! 
!     invoke-virtual {v3, v13, v0}, Lcom/android/internal/telephony/WapPushOverSms;->dispatchWapPdu([BLjava/lang/String;)I
  
      move-result v3
  
***************
*** 2260,2266 ****
  
      move-result-object v4
  
!     invoke-virtual {v3, v4}, Lcom/android/internal/telephony/WapPushOverSms;->dispatchWapPdu([B)I
  
      move-result v3
  
--- 2299,2307 ----
  
      move-result-object v4
  
!     move-object/from16 v0, p2
! 
!     invoke-virtual {v3, v4, v0}, Lcom/android/internal/telephony/WapPushOverSms;->dispatchWapPdu([BLjava/lang/String;)I
  
      move-result v3
  
